packages/runtime/container-runtime/src/test/gc/gcConfigs.spec.ts

```diff
@@ -440,16 +463,16 @@ describe("Garbage Collection configurations", () => {
 				"getMetadata returned different metadata than expected",
 			);
 		});
-		it("Metadata Roundtrip with only sweepGeneration", () => {
+		it("Metadata Roundtrip with only tombstoneGeneration", () => {
```

Comment:
Using _only_ `sweepGeneration` is not advised or supported - in later versions, the two generation options are combined.